stable/kommander/values.yaml

```diff
@@ -52,31 +52,9 @@ federate:
     tag: "v0.1.1"
     pullPolicy: IfNotPresent
 
-  systemNamespace:
-    enable: true
-    name: kommander-system
-
-  kubeaddons:
-    image:
-      repository: "mesosphere/kubeaddons"
-      tag: "v0.5.3"
-      pullPolicy: IfNotPresent
-
 kommander-cluster-lifecycle:
   enabled: true
 
-  certificates:
-    issuer:
-      name:
-      kind: Issuer
-      selfSigned: false
-  konvoy:
-    allowUnofficialReleases: false
-  kubeaddonsRepository:
-    versionStrategy: repository-tag  # One of: `repository-tag`, `mapped-kubernetes-version`, `master`,
```

Comment:
I'll miss these comments. Perhaps we have some documentation where they are explained :)

Reply:
They are in the original chart.